apprenda55-express/azuredeploy.json

```diff
@@ -0,0 +1,239 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01-preview/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "platformAdminFirstName": {
+            "type": "string",
+            "metadata": {
+              "Description": "This is the first name of the platform administrator."
+            }
+        },
+        "platformAdminLastName": {
+            "type": "string",
+            "metadata": {
+              "Description": "This is the last name of the platform administrator"
+            }
+        },
+        "platformAdminEmailAddress": {
+            "type": "string",
+            "metadata": {
+              "Description": "This is the email address of the platform administrator. This will be used to log in."
+            }
+        },
+        "platformAdminPassword": {
+            "type": "string",
+            "metadata": {
+              "Description": "This is the password of the platform administrator. This will be used to log in."
+            }
+        },
+        "newStorageAccountName": {
+            "type": "string",
+            "metadata": {
+              "Description": "This is the name of the storage account the user will place their VMs."
+            }
+        },
+        "adminUsername": {
+            "type": "string",
+            "metadata": {
+              "Description": "This is the username of the administrative account."
+            }
+        },
+        "adminPassword": {
+            "type": "securestring",
+            "metadata": {
+              "Description": "This is the password of the administrative account."
+            }
+        },
+        "location": {
+            "type": "string",
+            "metadata": {
+              "Description": "This is the name of the Availability Set"
+            }
+        },
+        "storageAccountType": {
+            "type": "string",
+            "allowedValues": [ "Standard_LRS", "Standard_GRS"],
+            "metadata": {
+              "Description": "This is the storage account type."
+            }
+        },
+        "vmSize": {
+            "type": "string",
+            "allowedValues": ["Standard_A0", "Standard_A1", "Standard_A2", "Standard_A3", "Standard_A4"],
+            "metadata": {
+              "Description": "The size of the virtual machine."
+            }
+        },
+        "vmName": {
+            "type": "string",
+            "metadata": {
+              "Description": "This is the name of the virtual machine."
+            }
+        },
+        "subscriptionId": {
+            "type": "string",
+            "metadata": {
+              "Description": "This is the user's subscription ID."
+            }
+        },
+        "publicIPAddressName": {
+            "type": "string",
+            "metadata": {
+              "Description": "This is the name of the public IP address provisioned."
+            }
+        },
+        "virtualNetworkName": {
```

Comment:
There's an opportunity to hardcode this into a variable and have your users enter less input parameters